cypress/features/regression/experimental/select.feature

```diff
@@ -1,5 +1,5 @@
-Feature: Select component
-  I want to change Select component properties
+Feature: Experimental Select component
+  I want to change Experimental Select component properties
```

Comment:
Test case with `size` should be uncommented for default story.

Reply:
fixed